pycoal/mineral.py

```diff
@@ -24,6 +24,42 @@
 import time
 import fnmatch
 import shutil
+from pycoal import mineral
+import multiprocessing
+from joblib import Parallel, delayed
```

Comment:
joblib needs to be added to [setup.py](https://github.com/capstone-coal/pycoal/blob/master/setup.py#L38)